### PR TITLE
Update to 1.4.6

### PR DESCRIPTION
**Pull request by alex285**
also move from JSON to YAML
also require elementary theme

@TingPing since none seems to maintain this, can i have push access please?

### CODE REVIEW

**Comment by TingPing** [comment]
This package was made by the author of the app, so I'll leave that up to @lainsce if they want help.

File: com.github.lainsce.notejot.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+id: com.github.lainsce.notejot
```
Comment (TingPing) [inline com.github.lainsce.notejot.yml]: Don't rewrite the entire manifest in the same commit that updates it...

**Comment by lainsce** [comment]
If someone's willing to update, they can update. I trust Alex on this.
But yeah, it should first update the version, then port it to YAML, since it seems doing both confuses the packaging bot.

**Comment by alex285** [comment]
@lainsce can i push the `.yml` with the old Notejot version, and then push again with the version update and the GTK patch?

**Comment by jurf** [comment]
> If someone's willing to update, they can update. I trust Alex on this.
> But yeah, it should first update the version, then port it to YAML, since it seems doing both confuses the packaging bot.

I don’t see why that would _confuse_ the bot… it’s not something the bot cares about, it’s more about us humans. :grinning:

**Comment by lainsce** [comment]
So it seems I commented while having a broken telephone on it.
It's ok to have it first be updated then YAML'd for my convenience.

**Pull request merged**